rero_ils/modules/acq_order_lines/mappings/v7/acq_order_lines/acq_order_line-v0.0.1.json

```diff
@@ -30,13 +30,19 @@
       },
       "document": {
         "properties": {
+          "type": {
```

Comment:
Add in the commit message the modification that you added to the `mapping` files

Reply:
done